pkgs/by-name/ol/olympus-unwrapped/update.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq common-updater-scripts
+
+set -eu -o pipefail
+
+attr=olympus.unwrapped
```

Comment:
```suggestion
attr=olympus-unwrapped
```
Please test the update script after you made changes.